### PR TITLE
Terse notes of 2023-11-30 board meeting

### PR DESCRIPTION
**Pull request by gruetter**
This PR adds the terse notes for our Nov. 30th board meeting.

### CODE REVIEW

File: content/en/about/board/2023-11-30.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+show_meta: false
+title: '2023-11-30'
+---
+
+### Date and Time of Meeting
+
+* Call to order: 2023-11-28, 2:07pm CET
```
Comment (tsadler1988) [inline content/en/about/board/2023-11-30.md]:
```suggestion
* Call to order: 2023-11-30, 2:07pm CET
```

**Review by tsadler1988 — CHANGES_REQUESTED** [review]
One small error but otherwise looks good!

**Comment by gruetter** [comment]
Thanks for the "bugfix", @tsadler1988 ! I have updated the doc.

**Review by tsadler1988 — APPROVED** [review]

**Comment by tsadler1988** [comment]
Thanks @gruetter

**Pull request merged**